### DISCUSSION TITLE
pt-20 and pt-48 only work

### DISCUSSION THREAD

**Discussion by aileenrose** [tailwindlabs/tailwindcss#8811] [Help] [answered]
My apologies for such a newbie issue.  I would like to use the class pt-40 or pt-36, but neither work.  We are using the defaultTheme, so I assume both should be available.  TIA!

**Comment by aileenrose** [comment, marked as answer]
User error. :) This is resolved.

**Comment by o-bardiuk** [comment]
For whoever googles this:

Check **tailwind.config.js**

There should be a section **spacing**
